lxd/api_project.go

```diff
@@ -520,10 +520,12 @@ func isEitherAllowOrBlockOrManaged(value string) error {
 func projectValidateConfig(s *state.State, config map[string]string) error {
```

Comment:
Can you rename this commit to `lxd/projects: XYZ` instead of `lxd/api: XYZ`?

Just a very minor nit ;)